Specs/Number/Korean/NumberModel.json

```diff
@@ -2911,7 +2883,7 @@
     "IgnoreResolution": true,
     "NotSupportedByDesign": "java, javascript, python",
     "NotSupported": "dotnet",
-    "Comment": "PendingImplementation",
+    "Comment": "As per language expert, input needs to be corrected",
```

Comment:
Can you explain why in the PR description?

Reply:
Sure.